### PR TITLE
#4 Configure CI/CD

### PR DESCRIPTION
**Pull request by mirogta**
creating a test PR to see if this works

### CODE REVIEW

**Comment by mirogta** [comment]
OK - the CI bit works...

Moving on to CD...

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      max-parallel: 3
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-18.04]
+        node-version: [8.x]
+        #node-version: [8.x, 10.x]
```
Comment (bkielbasa) [inline .github/workflows/ci.yml]: why not using the latest node version?

**Review by bkielbasa — APPROVED** [review]

**Comment by bkielbasa** [comment]
sorry, but I didn't notice the PR

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mirogta) [reply .github/workflows/ci.yml]: good point, I'll try with the latest - shouldn't make much difference just for the CI

**Comment by mirogta** [comment]
> sorry, but I didn't notice the PR

no problem - wanted to ping you once I put the CD bit there as well.

I'm looking at the deployment options, it is a bit more involved and I had to park this for a few days

**Comment by bkielbasa** [comment]
you can always add CD later.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mirogta) [reply .github/workflows/ci.yml]: updated with node 13.x - build still passing

**Pull request merged**